com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDAdditionalLightData.cs

```diff
@@ -2282,6 +2283,18 @@ private void Start()
             m_Animated = GetComponent<Animator>() != null;
         }
 
+        void Update()
```

Comment:
It is sad we must delay the needRefreshPrefabInstanceEmissiveMeshes refresh here:
Now, we must support a call from C++ to C# at each frame. So it is consuming resources (even if very few)
I expect small performance drawback fro scene with a lot of lights

Could it be first thing done in the already used LateUpdate?

Reply:
This seems to work for me (tests still green and warning does not repro any more in the test project).

Before merge, just to be safe, I think @TomasKiniulis should also take it for another spin with his repro project with emissive mesh prefabs.

Reply:
Looks all good to me in LateUpdate too. Checking selecting emissive mesh prefab instances/variants, entering their prefab mode, adding additional prefab instances/variants in prefab mode, not encountering the warning